### PR TITLE
Cleanup to get rid of a few compiler warnings.

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/XbaseRuntimeModule.xtend [hunk 1]
```diff
@@ -14,7 +14,7 @@ package org.eclipse.xtext.xbase
  */
 class XbaseRuntimeModule extends AbstractXbaseRuntimeModule {
 
-	@SuppressWarnings("deprecation") override Class<? extends org.eclipse.xtext.formatting.IFormatter> bindIFormatter() {
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/XbaseRuntimeModule.xtend]: should we have a look if we can delete that methods?

Comment (szarnekow) [inline org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/XbaseRuntimeModule.xtend]: Yes, that would be good. But I'd like to investigate this in a separate PR.

**Comment by szarnekow** [comment]
Is this ready to be merged? Any objections?

**Comment by kthoms** [comment]
LGTM

**Pull request merged**